ND_Supporting_Document_3_0.adoc

```diff
@@ -85,8 +85,8 @@ link:#_Toc412821715[Table 3: Evaluation Equivalency Analysis]
 
 [arabic, start=5]
 . Evaluation Activities can be defined for both Security Functional Requirements and Security Assurance Requirements (SAR). These are defined in separate sections of this Supporting Document.
-. If any Evaluation Activity cannot be successfully completed in an evaluation, then the overall verdict for the evaluation is a ‘fail’. In rare cases there may be acceptable reasons why an Evaluation Activity may be modified or deemed not applicable for a particular TOE, but this must be agreed with the Certification Body for the evaluation.
-. In general, if all Evaluation Activities (for both SFRs and SARs) are successfully completed in an evaluation then it would be expected that the overall verdict for the evaluation is a ‘pass’. To reach a ‘fail’ verdict when the Evaluation Activities have been successfully completed would require a specific justification from the evaluator as to why the Evaluation Activities were not sufficient for that TOE.
+. If any Evaluation Activity cannot be successfully completed in an evaluation, then the overall verdict for the evaluation is a ‘fail’. In rare cases there may be acceptable reasons why an Evaluation Activity may be modified or deemed not applicable for a particular TOE, but this must be agreed with the Certification Body for the evaluation and documented in the evaluation report.
+. In general, if all Evaluation Activities (for both SFRs and SARs) are successfully completed in an evaluation then it would be expected that the overall verdict for the evaluation is a ‘pass’.
 . Similarly, at the more granular level of Assurance Components, if the Evaluation Activities for an Assurance Component and all of its related SFR Evaluation Activities are successfully completed in an evaluation then it would be expected that the verdict for the Assurance Component is a ‘pass’. To reach a ‘fail’ verdict for the Assurance Component when these Evaluation Activities have been successfully completed would require a specific justification from the evaluator as to why the Evaluation Activities were not sufficient for that TOE.
```

Comment:
On #205, I actually pulled the text from the wrong paragraph! I've removed it from paragraph 8 as well now as I think the argument is valid for both.

Reply:
Test formatting comments:
Section 2.4.2.3, should these tests be cap letters? Should Test 1 be included? **I have adjusted these to include the standard introductory text and made these Test 1 and Test 2.**
Section 3.2.1.3, should test have a cap letter (B) to be consistent? **Since there is a wider description in para. 245, i.e. it expands on the normal, "The evaluator shall perform the following tests:", I left it as a para. number**
In Section 3.5.1.3, should tests be changed to Test 1a and 1b as opposed to Test 1.1 and 1.2? If so paragraph 272 would need edits. **Yes. Updated test names and the relevant references in para. 272**
FCS_TLSS_EXT.2.4 tests needs to be reformatted to capitol letters rather than paragraph numbers. And Cap letters to roman numerals **All done. Missed this when going through the reformatting of the tests**
FCS_DTLSS_EXT.1.3 and FCS_DTLSS_EXT.1.4, should C and D be roman numerals? **I've removed the bullets so that it matches the formatting in FCS_TLSS_EXT.1.2, i.e. it is just indented text as it relates to the test immediately before.**
FCS_DTLSS_EXT.1.6 and 1.7, should these tests have capitol letters rather than a paragraph #? **FCS_DTLSS_EXT.1.6 only has a single test so I've left it as a paragraph number. I suppose I could add the, "The evaluator shall perform..." text but its seemed unnecessary when there is only a single entry. I know that there are examples of this approach in the SD and can go fix. Thoughts? On 1.7, in my rendering, it has letters so that might be a Github ASCIIDOC issue**
FCS_TLSS_EXT.1.5 and 1.6, should these tests have capitol letters rather than a paragraph #. There may be additional examples of this earlier in the document that I missed. **As per the above, FCS_TLSS_EXT.1.5 is a single test so I left as a paragraph number. Again, I can change if you think it needs to follow the wider (new) convention.**
FCS_TLSS_EXT.1.2 and FCS_TLSS_EXT.1.3, should Test 2's 2nd and 3rd paragraphs have roman numerals for consistency? See FCS_DTLSS_EXT.1.3 and FCS_DTLSS_EXT.1.4. **As per the above. I've deliberately left these without bullets as the text relates to Test 2. So I've just kept them at the same indent level**
FCS_TLSS_EXT.1.5. and 1.6, should these tests have capitol letters rather than a paragraph #? **I think this is a repeat :)**
Test formatting should be reviewed for Sections 4.4.2.3 to the end of the SD. **I've checked these and updated as necessary. The one I haven't is the tests for FMT_MOF.1/Functions as the test structure there is a bit more complex and haven't decided have best to approach it (multiple iterations of Test 1, Test 2 etc.). I'll open a new issue for this specifically.**

Test reference comments:
FCS_DTLSS_EXT.1.9 test, paragraph 428, should test references be with a capitol I? Same question for FCS_TLSS_EXT.1.6.